src/components/pages/AdminHome/AdminHomeContent.js

```diff
@@ -15,11 +16,15 @@ const AdminHomeContent = () => {
       <div class="AdminDashHeaders">
         <div class="container courses">
           <h2>Recent Courses</h2>
-          <a href="/courses">See All</a>
+          <Link className="see-all-link" to="/admin-courses">
+            See All
+          </Link>
         </div>
         <div class="container applications">
           <h2>Instructor Applications</h2>
-          <a href="/instructors">See All</a>
+          <Link className="see-all-link" to="/admin-instructors">
+            See All
+          </Link>
```

Comment:
Changing things to include links speeds up navigation and stores destination to history to be more readily accessible.